go-controller/pkg/ovn/policy.go

```diff
@@ -73,6 +73,33 @@ const (
 	emptyLabelSelector  = "<none>"
 )
 
+func (oc *Controller) syncNetworkPolicies(networkPolicies []interface{}) {
+	expectedPolicies := make(map[string]map[string]bool)
+	for _, npInterface := range networkPolicies {
+		policy, ok := npInterface.(kapisnetworking.NetworkPolicy)
+		if !ok {
+			logrus.Errorf("Spurious object in syncNetworkPolicies: %v",
+				npInterface)
+			continue
+		}
+		expectedPolicies[policy.Namespace] = map[string]bool{
+			policy.Name: true}
+	}
+
+	err := oc.forEachAddressSet(func(addrSetName, asNamespace,
+		asName string) {
```

Comment:
asNamespace is better as namespaceName and asName is better as policyName.

Reply:
renamed asNamespace and asName to namespaceName and policyName.